plugins/include/amxconst.inc

```diff
@@ -29,6 +29,11 @@
  */
 #define MAX_STRING_LENGTH 16384
 
+/**
+ * The maximum buffer size required to store a map's name.
+ */
+#define MAX_MAPNAME_LENGTH 256
```

Comment:
Why 256? [It is limited to be 64](https://github.com/dreamstalker/rehlds/blob/237ee464644b5c8d9485c1e4943019a24ade76e1/rehlds/engine/server.h#L114).

Reply:
With a margin :D